usr.sbin/daemon/daemon.c

```diff
@@ -292,22 +301,15 @@ main(int argc, char *argv[])
 	 * and syslog.
 	 */
 	if (pidfile || ppidfile || restart || outfd != -1 || dosyslog) {
-		struct sigaction act_term, act_chld, act_hup;
```

Comment:
I have mixed feelings about this one, given that they're really only used within this block